lib/presentation/di/module/widget_module.dart

```diff
@@ -446,4 +449,10 @@ class WidgetModule {
         ));
   }
 
+  void _provideHomeAppBarComponent() {
+    getIt.registerFactory(() => HomeAppBarWidget());
```

Comment:
Registered `HomeAppBarWidget` but there is no usage of it with `getIt`

Reply:
you are right I just need the viewmodel